perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -30,6 +53,10 @@ class GCPManagedRelationalDb(managed_relational_db.BaseManagedRelationalDb):
 
   CLOUD = providers.GCP
   SERVICE_NAME = 'managed_relational_db'
+  # These are the constants that should be specified in GCP's cloud SQL command.
+  DEFAULT_BACKUP_START_TIME = '07:00'
+  GCP_MY_SQL_VERSION = 'MYSQL_5_6'
```

Comment:
Also, I would call this DEFAULT_GCP_MYSQL_VERSION because it should support being overridden in the managed_db_spec